### PR TITLE
Add Exemple for instance group tagging

### PR DESCRIPTION
**Pull request by sergeohl**
I had doc to show how to add tags on aws ASG and node

### CODE REVIEW

File: docs/instance_groups.md [hunk 1]
```diff
@@ -242,3 +242,26 @@ spec:
               - http://local-mirror.mydomain
               - http://archive.ubuntu.com
 ```
+
+## Add Tags on AWS autoscalling group and instance
```
Comment (chrislovecnm) [inline docs/instance_groups.md]: A nitpick that you can do in a follow-up PR or someone else can edit.  These labels should work in GCE as well now.

Comment (sergeohl) [reply docs/instance_groups.md]: When I check the godocs (https://godoc.org/k8s.io/kops/pkg/apis/kops#InstanceGroupSpec), there is just mentin of AWS

```// CloudLabels indicates the labels for instances in this group, at the AWS level
    CloudLabels map[string]string `json:"cloudLabels,omitempty"```

Comment (chrislovecnm) [reply docs/instance_groups.md]: I am pretty certain the godoc are out of date :)  We are good for now.

**Pull request merged**